instrument/EXED_Definition.xml

```diff
@@ -0,0 +1,590 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<instrument name="EXED" date="6th Mar 2017"
```

Comment:
I matched the date format to other instrument definition files and updated the pixel sizes appropriately